user/src/components/LoginForm/login_form.jsx

```diff
@@ -0,0 +1,130 @@
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useForm } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+import * as yup from "yup";
+import { toast, ToastContainer } from 'react-toastify';
+
+
+const LoginForm = ({activeTab}) => {
+  const navigate = useNavigate();
+  const Schema = yup
+  .object({
+    email: yup.string().required("Email không được để trống"),
+    password: yup
+    .string()
+    .required("Mật khẩu không thể để trống")
+  })
+  .required();
+
+  const {
+    register,
+    handleSubmit,
+    formState: {errors},
+  } = useForm({
+    mode: "onSubmit",
+    defaultValues: {
+      email: "",
+      password: "",
+    },
+    resolver: yupResolver(Schema),
+  });
+
+  const handleLogin = async (data) => {
+    const url = process.env.REACT_APP_HOST + `/users?email=${data.email}&password=${data.password}`
+    await fetch(url)
+    .then(res => res.json())
+    .then((user) => {
+      if (Object.keys(user).length === 0) {
```

Comment:
```suggestion
      if (user.length === 0) {
```
`user` này là mảng anh nhỉ